src/main/java/kitchenpos/domain/Order.java

```diff
@@ -1,64 +1,135 @@
+
+import org.springframework.util.CollectionUtils;
+
+@Entity
+@Table(name = "orders")
 public class Order {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long id;
-	private Long orderTableId;
-	private String orderStatus;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "order_table_id")
+	private OrderTable orderTable;
+
+	@Enumerated(value = EnumType.STRING)
+	private OrderStatus orderStatus;
+
+	@Column(name = "ordered_time")
 	private LocalDateTime orderedTime;
-	private List<OrderLineItem> orderLineItems;
+
+	@OneToMany(mappedBy = "order")
+	private List<OrderLineItem> orderLineItems = new ArrayList<>();
 
 	protected Order() {
 	}
 
-	public Order(Long id, Long orderTableId, String orderStatus, LocalDateTime orderedTime,
+	public Order(OrderTable orderTable, OrderStatus orderStatus, LocalDateTime orderedTime,
 		List<OrderLineItem> orderLineItems) {
-		this.id = id;
-		this.orderTableId = orderTableId;
+
+		validateEmptyTableOrder(orderTable);
+		ValidateEmptyOrderItems(orderLineItems);
+
+		for (OrderLineItem orderLineItem : orderLineItems) {
+			addOrderLineItem(orderLineItem);
+		}
+		order(orderTable);
 		this.orderStatus = orderStatus;
 		this.orderedTime = orderedTime;
-		this.orderLineItems = orderLineItems;
 	}
 
-	public Long getId() {
-		return id;
+	public void order(OrderTable orderTable) {
+		this.orderTable = orderTable;
+		orderTable.addOrder(this);
 	}
 
-	public void setId(final Long id) {
-		this.id = id;
+	private void ValidateEmptyOrderItems(List<OrderLineItem> orderLineItems) {
+		if (CollectionUtils.isEmpty(orderLineItems)) {
+			throw new IllegalArgumentException("주문 항목을 구성해야 주문이 가능합니다.");
+		}
 	}
 
-	public Long getOrderTableId() {
-		return orderTableId;
+	private void validateEmptyTableOrder(OrderTable orderTable) {
+		if (orderTable.isEmpty()) {
+			throw new IllegalArgumentException("비어있는 테이블은 주문할 수 없습니다.");
+		}
 	}
 
-	public void setOrderTableId(final Long orderTableId) {
-		this.orderTableId = orderTableId;
+	public void addOrderLineItem(OrderLineItem orderLineItem) {
+		this.orderLineItems.add(orderLineItem);
+		orderLineItem.changeOrder(this);
 	}
 
-	public String getOrderStatus() {
-		return orderStatus;
+	public OrderTable getOrderTable() {
+		return orderTable;
 	}
 
-	public void setOrderStatus(final String orderStatus) {
-		this.orderStatus = orderStatus;
+	public OrderStatus getOrderStatus() {
+		return orderStatus;
 	}
 
 	public LocalDateTime getOrderedTime() {
 		return orderedTime;
 	}
 
-	public void setOrderedTime(final LocalDateTime orderedTime) {
-		this.orderedTime = orderedTime;
-	}
-
 	public List<OrderLineItem> getOrderLineItems() {
 		return orderLineItems;
 	}
 
-	public void setOrderLineItems(final List<OrderLineItem> orderLineItems) {
-		this.orderLineItems = orderLineItems;
+	public Long getId() {
+		return id;
+	}
+
+	private void validateChangeStatus() {
+		if (Objects.equals(OrderStatus.COMPLETION.name(), this.getOrderStatus().name())) {
```

Comment:
```suggestion
		if (orderStatus.isCompletion()) {
```
enum도 객체입니다. 완료 여부를 체크하는 메서드를 추가해보아요